tests/scripts/custodian/user-keys.txt

```diff
@@ -0,0 +1,15 @@
+ctw
```

Comment:
This is going to need a PR every time this need a change.

Im more inclined on adding a Jenkins parameter with the current keys using the job builder. Or something not coming from `rancher/rancher`

Reply:
that's a good idea, I will address this once I get more feedback from others on the rest of the PR.

Reply:
I like Izaac's suggestion here! It will help a lot during code-freeze periods.

Reply:
Let's make this change prior to merging for the dry-run

Reply:
removed from the file, will add a jenkins parameter once the job PR merges